content-review/impl/turnitin-oc/src/main/resources/turnitin-oc.properties

```diff
@@ -35,9 +35,9 @@ report.error.unknown=Unknown error occurred retrieving an originality report
 
 submission.terminal.status.UNSUPPORTED_FILETYPE=The uploaded filetype is not supported
 submission.terminal.status.PROCESSING_ERROR=An unspecified error occurred while processing the submissions
-submission.terminal.status.TOO_LITTLE_TEXT=The submission does not have enough text to generate a Similarity Report (a submission must contain at least 20 words)
```

Comment:
Why are the useful informational bits being removed?

Reply:
These details are too specific. Each fact has already changed and is incorrect. It seems odd to have to maintain these in a properties file

Reply:
So these limits are no longer the case?

Reply:
sorry, looks like only one has changed. Too little text is the same, too many pages is 800 now, too much text is still 2MB

Reply:
(this is for TurnitinOC)

Reply:
Thanks for clarifying, @baholladay. Would it not be useful to still communicate this with end-users through these messages?

Reply:
I'll re-add the specific wording... I'll have to re-submit and retranslate so it'll be a few

Reply:
Thanks, @baholladay!

Reply:
updated! please review and merge

Reply:
actually deleting this PR and opened a new one: https://github.com/sakaiproject/sakai/pull/8228